home/gettingstarted/0.7/persistors.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: docsdefault07
+title: Persistors
+permalink: /home/gettingstarted/0.7/persistors/index.html
+
+smversion: 0.7
+partof: getting-started
+num: 6
+outof: 50
+---
+
+
+`Persistor`s are used to store results from executed benchmarks.
+
+## Persistor.None
+
+It actually does no persisting at all.
+
+## SerializationPersistor
+
+It uses `Java serialization` under the hood.  
+
+Note that this persistor is not a good choice if you want to keep you results while upgrading `ScalaMeter` version.
+
+## JSONSerializationPersistor
+
+It writes a regression data to disk as JSON. 
+
+It's the fastest among `Persistor`s, although it produces the largest output files.   
+
+## GZIPJSONSerializationPersistor
+
+Default choice for all `HTMLReport` descendants. 
+
+It works like `JSONSerializationPersistor`, but instead of storing a plain JSON on disk it gzips it firsts.
+This allows to be much more space efficient than `JSONSerializationPersistor` 
+whilst being a little slower (but still faster than `Serialization Persistor`).
+
+## Pickler
+
+For JSON-based persistors `Parameters` are stored as Base64-encoded strings. 
+This enforces that every `Parameter` (and thus `Gen` implementation) requires `Pickler`. 
+
+We provide `Pickler`s for most commonly using types, 
+which are avaialable when importing `org.scalameter.picklers.Implicits._`.
+
+However, if you would like to make own generator or e.g. invoke `Gen.enumeration` with custom class 
+you must either write `Pickler` for that class or import `org.scalameter.picklers.noPickler._` and use `SerializationPersistor`.
```

Comment:
then you must either implement a `Pickler` for that class.

If you don't want to implement a `Pickler` for your custom class, then you can import `org.scalameter.picklers.noPickler._`. You will not have to implement a custom `Pickler`, but you will only be able to use the `SerializationPersistor`.